app/controllers/api/v2/organisations_controller.rb

```diff
@@ -6,11 +6,12 @@ def index
         @organisations = Organisation.all
         if params_includes_provider?
           current_recruitment_cycle = RecruitmentCycle.current
-          @organisations = @organisations.includes(:providers)
+          @organisations = @organisations.includes(:providers, :users, :nctl_organisations)#, providers: %i[contacts sites courses recruitment_cycle ucas_preferences])
```

Comment:
Why comment out `providers: %i[contacts sites courses recruitment_cycle ucas_preferences`?

Reply:
Typo?

Reply:
That was all the stuff being lazy loaded that I was trying to get rid of by frontloading it before realising it didn't help.